tests/translator/test_translator.py

```diff
@@ -271,12 +287,29 @@ def test_transform_success(self, testcase, partition_with_region):
 
         print(json.dumps(output_fragment, indent=2))
 
+        # Run cfn-lint on translator test output files.
+        rules = cfnlint.core.get_rules([], LINT_IGNORE_WARNINGS, [])
+
         # Only update the deployment Logical Id hash in Py3.
         if sys.version_info.major >= 3:
             self._update_logical_id_hash(expected)
             self._update_logical_id_hash(output_fragment)
+            output_template = cfnlint.decode.cfn_json.load(expected_filepath)
+        else: # deprecation warning catching in py2
+            import warnings
```

Comment:
Is it good python style to use import in the middle of the code? Usually those things are at the top of the file I think

Reply:
Usually they are, but I feel it's fine in this case because we are only using it in py2.